CardanoSharp.Wallet/AddressService.cs

```diff
@@ -10,10 +10,15 @@ namespace CardanoSharp.Wallet
 {
     public interface IAddressService
     {
+        [Obsolete]
         Address GetAddress(PublicKey payment, PublicKey stake, NetworkType networkType, AddressType addressType);
+        Address GetBaseAddress(PublicKey payment, PublicKey stake, NetworkType networkType);
+        Address GetRewardAddress(PublicKey stake, NetworkType networkType);
+        Address GetEnterpriseAddress(PublicKey payment, NetworkType networkType);
```

Comment:
@safestak-keith please see these three new methods. I also marked the original method as obsolete. we will retire that method later.